tests/func/test_add.py

```diff
@@ -168,17 +168,19 @@ def test(self):
         self.assertTrue(stages[0] is not None)
 
 
-def test_add_file_in_dir(repo_dir, dvc_repo):
-    stage, = dvc_repo.add(repo_dir.DATA_SUB)
+def test_add_file_in_dir(dvc, tmp_dir):
```

Comment:
The order doesn't matter to python, but it matters to reader. Let's list fixtures in the same order everywhere: `tmp_dir`, `scm` (if used), `dvc`. This is chronological: temp dir is creared, then git init, then dvc init.